packages/code-gen/package.json

```diff
@@ -1,7 +1,16 @@
 {
-  "name": "code-gen",
-  "version": "0.0.1",
+  "name": "@dynodm/code-gen",
+  "version": "0.0.0",
```

Comment:
```suggestion
  "version": "0.0.1",
```

Reply:
Will be bumped automatically by lerna when publishing :)